examples/text_to_image/train_text_to_image_sdxl.py

```diff
@@ -572,6 +572,15 @@ def generate_timestep_weights(args, num_timesteps):
     return weights
 
 
+def print_gpu_utilization(idx=0):
```

Comment:
we probably don't need to add this function, no?

Reply:
yes